h2o-py/h2o/model/permutation_varimp.py

```diff
@@ -26,6 +26,12 @@ def permutation_varimp(model, frame, use_pandas=True, metric="mse"):
     if type(frame) is not H2OFrame:
         raise ValueError("Frame is not H2OFrame")
 
+    # Check if metric exists for the model
```

Comment:
We talked about getting the available metrics from json. For GBM it works, however, for standalone_test(perm_var_imp_plot_glm) on pyunit_pubdev_7139_permutation_var_imp.py. The json format doesn't show all the metrics. Not even MSE nor logloss which this model's family is Binomial

Reply:
I will take a look.

Reply:
Try look into training metrics instead of into scoring history.  For example this command `model._model_json['output']['training_metrics']._metric_json` returns list of available metrics.

Reply:
But be careful, the metrics there have concrete names for example MSE instead of mse.